ios/FluentUI/Notification/NotificationView_SwiftUI.swift

```diff
@@ -172,47 +168,30 @@ public struct NotificationViewSwiftUI: View {
     }
 
     public var body: some View {
-        let width = UIScreen.main.bounds.width - safeAreaInsets.leading - safeAreaInsets.trailing
-        innerContents
-            .onTapGesture {
-                if state.messageButtonAction != nil {
-                    activeAlert = .message
-                    showingAlert = true
-                }
-            }
-            .frame(width: state.style.isToast && horizontalSizeClass == .regular ? width / 2 : width - (2 * tokens.presentationOffset))
-            .background(
-                RoundedRectangle(cornerRadius: tokens.cornerRadius)
-                    .strokeBorder(Color(tokens.outlineColor), lineWidth: tokens.outlineWidth)
-                    .background(
-                        RoundedRectangle(cornerRadius: tokens.cornerRadius)
-                            .fill(Color(tokens.backgroundColor))
-                    )
-                    .shadow(color: Color(tokens.ambientShadowColor), radius: tokens.ambientShadowBlur, x: tokens.ambientShadowOffsetX, y: tokens.ambientShadowOffsetY)
-                    .shadow(color: Color(tokens.perimeterShadowColor), radius: tokens.perimeterShadowBlur, x: tokens.perimeterShadowOffsetX, y: tokens.perimeterShadowOffsetY)
-            )
-            .designTokens(tokens,
-                          from: theme,
-                          with: windowProvider)
-            .alert(isPresented: $showingAlert) {
-                var alertString: String
-                switch activeAlert {
-                case .actionButton:
-                    if let actionTitle = state.actionButtonTitle {
-                        if actionTitle.isEmpty {
-                            alertString = "Dismiss"
-                        } else {
-                            alertString = actionTitle
-                        }
-                    } else {
-                        alertString = "Action"
+        if let windowWidth = UIApplication.shared.keyWindow?.bounds.width {
```

Comment:
Have you looked at the GeometryReader to retrieve the dimensions of the screen?
I think using UIKit's window is not ideal here.

Reply:
As discussed offline, we will keep the UIKit's window usage here because GeometryReader relies on the bounds of the hosting controller which cannot be set accurately at this point

Reply:
Mmmmmm I hear your point about not being able to use `GeometryReader`, but I think there's still a real scenario here for multi-window. Example: what if I have two Word windows visible, am actively typing in one (making it `keyWindow`), but the other window is trying to show a notification?